docs/Android.md

```diff
@@ -161,5 +161,5 @@ $ utils/build-script \
   --android \                        # Build for Android.
   --android-ndk ~/android-ndk-r23b \  # Path to an Android NDK.
   --android-arch armv7 \             # Optionally specify Android architecture, alternately aarch64
-  --android-ndk-version 21
```

Comment:
There is no such flag, which [was corrected in the build script but missed in this doc](https://github.com/apple/swift/pull/20805/files#diff-9a1726f5e9615fbcad51fcaead5b79cb33f26e6685f4871910e1582ae9049147R257).